src/Scynet.HatcheryFacade/appsettings.json

```diff
@@ -4,5 +4,12 @@
       "Default": "Warning"
     }
   },
-  "AllowedHosts": "*"
+  "AllowedHosts": "*",
+  "RPC": {
+    "port": 9998,
+    "ip": "0.0.0.0"
+  },
+  "Kafka": {
+    "brokers": [ "127.0.0.1:9092" ]
+  } 
```

Comment:
Configuration is unused in this PR.
Also, `brokers` should be with a capital letter.